pages/linux/lxc-checkconfig.md

```diff
@@ -0,0 +1,12 @@
+# lxc-checkconfig
+
+> Check a kernel for LXC support.
+> More information: <https://linuxcontainers.org/lxc/getting-started/>.
+
+- Check the current kernel for LXC support:
+
+`lxc-checkconfig`
+
+- Don't show informative messages:
+
+`CONFIG=/path/to/configuration initramfs-linux-zen.img`
```

Comment:
Is this connect? This does not seem like a command.

Reply:
You can set temporary environment variables before a command on the same line.

Reply:
Yes, but the command is `initramfs-linux-zen.img` in this case. Is this correct? If so, how does it relate to lxc-checkconfig? I cannot find it in the more information link

Reply:
My bad, I must have pasted the wrong part.

Reply:
Also, the command seems to be setting a config file. Does this file need to exist? Or can it be a placeholder?

Reply:
I added a placeholder. Creating pages while tired and watching a streamer is a mistake.